### DISCUSSION TITLE
Simplepyble service/characteristics visibility

### DISCUSSION THREAD

**Discussion by MatejHaronik** [simpleble/simpleble#372] [Q&A]
Hello all,
im using simplepyble in visual studio and stm32 wb55cg micro. I managed to connect to device. I can see my service broadcast before conection to device, but after connection im not able to acces service.

**Comment by kdewald** [comment]
Hey @MatejHaronik, thanks for writing. Can you reach out to `contact@simpleble.org` for us to look into this?
